packages/core/.babelrc.js

```diff
@@ -1,12 +0,0 @@
-const { babelOptimizerPlugin } = require('@graphql-codegen/client-preset')
```

Comment:
Should we remove this lib from `package.json`?

Reply:
When I tried to remove it and perform the `generate:codegen` script I got an error and I assumed it's required even without being used by the Babel optimizer, but I'll confirm it again

Reply:
<img width="820" alt="Screenshot 2024-05-14 at 21 56 47" src="https://github.com/vtex/faststore/assets/15722605/65fe0315-7062-4a20-94e6-687c88c31a5d">